components/clinical/shared/type-detail/src/molecules/string-detail.tsx

```diff
@@ -3,14 +3,24 @@ import ReactHtmlParser from 'react-html-parser'
 import { DetailViewComponent, IDetailViewProps } from '../atoms/detail-view-component'
 import NestedListDetail from './nested-list-detail'
 
-const StringDetail: DetailViewComponent<IProps> = ({ term, description, showIfEmpty }) => {
+const StringDetail: DetailViewComponent<IProps> = ({ term, description, showIfEmpty, parse = true }) => {
   if (description || showIfEmpty) {
+    if (parse) {
+      return (
+        <NestedListDetail
+          term={term}
+          className={(description ?? '').length > 150 ? 'string-detail--full-width' : 'string-detail'}
+        >
+          {ReactHtmlParser(titleCase(description ?? ''))}
+        </NestedListDetail>
+      )
+    }
     return (
       <NestedListDetail
         term={term}
         className={(description ?? '').length > 150 ? 'string-detail--full-width' : 'string-detail'}
       >
-        {ReactHtmlParser(titleCase(description ?? ''))}
+        {description ?? ''}
```

Comment:
a ternary here would save repeated code e.g.

```javascript
{parse === true ? ReactHtmlParser(titleCase(description ?? '') : description ?? ''}
```

Reply:
You wouldn't believe how long I tried fiddling with brackets to make it work like that initially! You've encouraged me to give it another go and I've cracked it now - thanks!

Reply:
Updated...